flight/Modules/Telemetry/telemetry.c

```diff
@@ -970,12 +958,7 @@ static void session_managing_updated(UAVObjEvent * ev, void *ctx, void *obj, int
  */
 static void update_object_instances(uint32_t obj_id, uint32_t inst_id)
 {
-	SessionManagingData sessionManaging;
-	SessionManagingGet(&sessionManaging);
-	sessionManaging.ObjectID = obj_id;
-	sessionManaging.ObjectInstances = inst_id;
-	sessionManaging.SessionID = sessionManaging.SessionID + 1;
-	SessionManagingSet(&sessionManaging);
+	/* XXX just send the new instance, reqack. */
```

Comment:
Leaving this as todo, because it's just not necessary right now-- the new instances get telemetered, we hear them, and know about them.  When doing further work on waypoints this might become helpful.